solc/CommandLineInterface.cpp

```diff
@@ -541,6 +548,7 @@ Allowed options)",
 			"Estimated number of contract runs for optimizer tuning."
 		)
 		(g_argAddStandard.c_str(), "Add standard contracts.")
+		(g_argPrettyJson.c_str(), "Output JSON in pretty format.")
```

Comment:
It is a little confusing that this only applies to `--combined-json` and not to `--standard-json`.

Reply:
Hm, actually that is not controlled by the CLI, but is built in standarcompiler.

Reply:
Ok, but `--help` should explain that.

Reply:
We could change the API of StandardCompiler to support that if needed.

Reply:
Okay changed it for now, let's implement support for StandardCompiler separately.

Reply:
Is there anything you want me to do?

Reply:
Thanks for the contribution, it is now merged :)